pkg/validate/validate_values_test.go

```diff
@@ -32,6 +32,15 @@ func TestValidateValues(t *testing.T) {
 		{
 			desc: "nil success",
 		},
+		{
+			desc: "AsteriskIPRange",
+			yamlStr: `
```

Comment:
Should also test "*, 1.2.3.4" as an error case.

Reply:
Added below, thanks.